### DISCUSSION TITLE
Codespaces Not Loading

### DISCUSSION THREAD

**Discussion by TechItAllSA** [community/community#110509] [Codespaces]
My Codespace wont load and I keep getting the error "ECONNABORTED: (signal aborted) canceled" or "ECONNABORTED: (timeout) timeout of 20000ms exceeded". It Will load into it and be stuck on connecting to remote codespace for a bit then crash with that error message. I even made a new codespace and got the same issue. I'm not sure how to fix this. I am connected to the internet and others sites load fine. I have checked github status and it says they are online. I Use github on my managed chromebook. Im not sure if thats the reason why it blocked tho.

EDIT 1: I would also like to say that when my Codespace stopped working, my friends worked just fine. Same Wi-Fi, same computer. After a few weeks later his also stopped working.

**Comment by sergio-sak** [comment]
I have the same issue, did you find a solution?

**Reply by kuciapakrystian** [reply]
If you're encountering issues with GitHub Codespaces on your managed Chromebook, there could be a few potential causes and troubleshooting steps you can take:

Check for Chrome OS Updates: Ensure your Chromebook is running the latest version of Chrome OS. Sometimes, system updates can resolve compatibility issues.

Network Connectivity: Even though you're connected to the internet, Codespaces may have specific network requirements. Ensure that there are no firewall or proxy settings blocking access to GitHub services. You may want to try accessing GitHub Codespaces from a different network to see if the issue persists.

Browser Compatibility: GitHub Codespaces is designed to work best with the latest versions of modern browsers. Ensure that you're using a compatible browser and try accessing Codespaces from an alternative browser to see if the issue persists. Google Chrome is usually the recommended browser for Chromebook users.

Clear Browser Cache and Cookies: Sometimes, cached data or cookies can cause unexpected behavior. Try clearing your browser's cache and cookies and then attempt to access GitHub Codespaces again.

Extensions and Add-ons: Disable any browser extensions or add-ons that might interfere with GitHub Codespaces. Sometimes, ad blockers or security extensions can cause conflicts.

Local System Configuration: Check if there are any system settings or configurations on your managed Chromebook that could be causing the issue. If possible, try accessing GitHub Codespaces from a different device to see if the problem persists.

GitHub Status Page: Keep an eye on the GitHub Status Page for any reported issues or outages with GitHub services. Even though the status page may indicate that GitHub is online, there could still be localized issues affecting certain users or regions.

Contact GitHub Support: If none of the above steps resolve the issue, consider reaching out to GitHub Support for further assistance. They may be able to provide additional troubleshooting steps or insights specific to your situation.

By following these steps, you should hopefully be able to identify and resolve the issue with GitHub Codespaces on your managed Chromebook.

**Reply by willgamerthehcaker123** [reply]
me too when I get to school it is not loading codespace

**Comment by amankr1469** [comment]
Clear Chrome cache and cookies or try incognito mode to rule out extension interference.

**Reply by TechItAllSA** [reply]
Do you think it could be "Lightspeed Filter Agent"? That is what the school uses to block everything. But it never bring up the block screen and I'm able to use the rest of GitHub. They also disabled incognito mode.

**Comment by kuciapakrystian** [comment]
If you're encountering issues with GitHub Codespaces on your managed Chromebook, there could be a few potential causes and troubleshooting steps you can take:

Check for Chrome OS Updates: Ensure your Chromebook is running the latest version of Chrome OS. Sometimes, system updates can resolve compatibility issues.

Network Connectivity: Even though you're connected to the internet, Codespaces may have specific network requirements. Ensure that there are no firewall or proxy settings blocking access to GitHub services. You may want to try accessing GitHub Codespaces from a different network to see if the issue persists.

Browser Compatibility: GitHub Codespaces is designed to work best with the latest versions of modern browsers. Ensure that you're using a compatible browser and try accessing Codespaces from an alternative browser to see if the issue persists. Google Chrome is usually the recommended browser for Chromebook users.

Clear Browser Cache and Cookies: Sometimes, cached data or cookies can cause unexpected behavior. Try clearing your browser's cache and cookies and then attempt to access GitHub Codespaces again.

Extensions and Add-ons: Disable any browser extensions or add-ons that might interfere with GitHub Codespaces. Sometimes, ad blockers or security extensions can cause conflicts.

Local System Configuration: Check if there are any system settings or configurations on your managed Chromebook that could be causing the issue. If possible, try accessing GitHub Codespaces from a different device to see if the problem persists.

GitHub Status Page: Keep an eye on the GitHub Status Page for any reported issues or outages with GitHub services. Even though the status page may indicate that GitHub is online, there could still be localized issues affecting certain users or regions.

Contact GitHub Support: If none of the above steps resolve the issue, consider reaching out to GitHub Support for further assistance. They may be able to provide additional troubleshooting steps or insights specific to your situation.

By following these steps, you should hopefully be able to identify and resolve the issue with GitHub Codespaces on your managed Chromebook.

**Reply by TechItAllSA** [reply]
Thanks, I will try those. I'm not too sure why this could be happening tho. It used to work just fine.

**Comment by purpleguy5728** [comment]
I'm having the  same issue

**Comment by Cl00m** [comment]
same issue
on chrome and edge it‘s "it seems you are offline", 
on firefox it's "ECONNABORTED: (signal aborted) canceled"

**Reply by TechItAllSA** [reply]
is your PC or WiFi managed?

**Reply by Cl00m** [reply]
my network environment is controlled by school, but I believe it's ok, cause codespace works fine when I use vscode desktop,  
I noticed many cros request error as **CORS request did not succeed**, alone with cookies rejected as**<Error><Message>The requested resource does not support http method 'GET'.</Message></Error>**
I thought the server isn't working well

**Comment by ziqidella** [comment]
Having the same issue, how did you solve it?

**Comment by ryanlua** [comment]
I have the same issue on my school-issued Chromebook. It's a shame because I teach programming and I can't use GitHub Codespaces and instead need to use some wonky web editor since you can't install anything on school Chromebooks.

**Reply by sergio-sak** [reply]
My friend fixed it for me by changing servers, give it a try

**Reply by 54145a** [reply]
> My friend fixed it for me by changing servers, give it a try

It worked! Oooooooooooooooooh ty sooooooooooooooooo much :)

**Reply by ryanlua** [reply]
> My friend fixed it for me by changing servers, give it a try

Unfortunately, this didn't work for me—it's still the same issue. `ECONNABORTED: (signal aborted) canceled`

**Reply by purpleguy2423** [reply]
didn't work

**Comment by Arche1987** [comment]
cov[]()

**Comment by purpleguy2423** [comment]
does anyone know any other coding websites (except Replit because it sucks)
